change MAINTAINER to LABEL

utf18 opened the pull request:
i changed maintainer to label, since the former is deprecated.
https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

All the best,
Benjamin

<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

ben-st committed:
change MAINTAINER to LABEL

i changed maintainer to label, since the former is deprecated.
https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

All the best,
Benjamin

wd256 committed:
Fix PutObject/CopyObject with metadata for GCS gateway (#5828)

Make sure to apply standard headers such as Content-Type,
Content-Disposition and Content-Language to the correct
GCS object attributes during object upload and copy operations.

Fixes: #5800

harshavardhana reviewed (dismissed):
LGTM

nitisht reviewed (commented):
Thanks for the PR @ben-st . Could you pls also update https://github.com/minio/minio/blob/master/Dockerfile.release & https://github.com/minio/minio/blob/master/Dockerfile.dev

sinhaashish committed:
Introduce new unauthenticated endpoint /metric (#5723) (#5829)

/metric exposes Promethus compatible data for scraping metrics

Fixes: #5723

Commit:
do not set response header X-Frame-Options for S3 requests (#5838)

This change removes the X-Frame-Options header - It should
not be set for S3 requests since it can break CORS.

Fixes #5813

minio-trusted committed:
Update yaml files to latest version RELEASE.2018-04-19T22-54-58Z

wd256 committed:
Disable "chunked" uploading by the GCS client for objects smaller than the chunk size. (#5835)

By disabling chunked uploading when the object size is less than the chunk size,
memory is not allocated unnecessarily.

ebozduman committed:
Remove panic() and handle it appropriately (#5807)

This is an effort to remove panic from the source. 
Add a new call called CriticialIf, that calls LogIf and exits. 
Replace panics with one of CriticalIf, FatalIf and a return of error.

balamurugana committed:
use package name correctly (#5827)

harshavardhana committed:
Cleanup and make a safer code (#5794)

utf18 committed:
change maintainer to label for dev and realease
